cassandra/metrics.go

```diff
@@ -0,0 +1,42 @@
+package cassandra
+
+import (
+	"fmt"
+
+	"github.com/gocql/gocql"
+	"github.com/raintank/met"
+)
+
+type Metrics struct {
+	cassErrTimeout         met.Count
+	cassErrTooManyTimeouts met.Count
+	cassErrConnClosed      met.Count
+	cassErrNoConns         met.Count
+	cassErrUnavailable     met.Count
+	cassErrOther           met.Count
+}
+
+func (m *Metrics) Init(component string, stats met.Backend) {
```

Comment:
Why have an Init() function?  Why not just use the more familiar approach of a "New" function.
eg
```
func NewMetrics(component string, stats met.Backend) *Metrics {
}
```